lib/elinux_build_target.dart

```diff
@@ -371,10 +372,14 @@ class NativeBundle {
           '-DCMAKE_C_COMPILER_TARGET=$targetCompilerTriple',
         if (targetCompilerTriple != null)
           '-DCMAKE_CXX_COMPILER_TARGET=$targetCompilerTriple',
+        if (targetCompilerFlags != null) '-DCMAKE_C_FLAGS=$targetCompilerFlags',
+        if (targetCompilerFlags != null)
+          '-DCMAKE_CXX_FLAGS=$targetCompilerFlags',
+        '-DCMAKE_C_COMPILER=clang',
+        '-DCMAKE_CXX_COMPILER=clang++',
         eLinuxDir.path,
       ],
       workingDirectory: outputDir.path,
-      environment: <String, String>{'CC': 'clang', 'CXX': 'clang++'},
```

Comment:
This patch looks good to me. But could you please tell me the motivation of these changes?

Reply:
Because it has smaller influence range. If environment variable is set, all tools invoked by CMake will have this environment variable. If we use `-D`, it won't pollute anything.

Reply:
OK, thanks.